<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowGrid"/>
   <addaction name="actionSnapGrid"/>
   <addaction name="separator"/>
   <addaction name="actionSelection"/>
   <addaction name="actionBranches"/>
   <addaction name="actionFans"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
   <property name="toolTip">
    <string>Enable/Disable Grid</string>
   </property>
  </action>
  <action name="actionSnapGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Snap to Grid</string>
   </property>
   <property name="toolTip">
    <string>Snap Items to Grid</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
   <property name="toolTip">
    <string>Select Items</string>
   </property>
  </action>
  <action name="actionBranches">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Branches</string>
   </property>
   <property name="toolTip">
    <string>Add Branches</string>
   </property>
  </action>
  <action name="actionFans">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fans</string>
   </property>
   <property name="toolTip">
    <string>Add Fans</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
